src/zenml/cli/__init__.py

```diff
@@ -848,6 +848,43 @@
 zenml stack register-secrets [<STACK_NAME>]
 ```
 
+Administering your Code Repositories
+------------------------------------
+
+Code repositories enable ZenML to keep track of the code version that you use
+for your pipeline runs. Additionally, running a pipeline which is tracked in
+a registered code repository can speed up the Docker image building for
```

Comment:
```suggestion
a registered code repository can decrease the time it takes Docker to build images for
```

an alternative phrasing?